### PR TITLE
feat/prettier

### PR DESCRIPTION
**Pull request by dottharun**
**Description**

This PR adds Prettier to the Project.

### Todo:

- [x] - add CI check for Prettier.
- [x] - [blocked] trigger prettier from lint-staged. 
          - requires lint-staged #5680 .

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 02595b72413ddc63c049330781eb488f9f68d83b at: https://669aea0d941377c18cf1e0d5--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit eb4a36bdec4b1f199da3c3c23f855eaa9448083d at: https://669c29ea1a7db43ce79ec790--layer5.netlify.app

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "semi": true,
+  "tabWidth": 2,
+  "printWidth": 100,
+  "singleQuote": true,
```
Comment (leecalcote) [inline .prettierrc:5]: Am I a terrible person for wanting this to be false? 

Double quotes is the convention that we've been attempting to follow. If we change this to true, perhaps, the amount of churn (change) in this PR will drop a bit.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: .prettierrc (hunk 1 above)
Comment (dottharun) [reply .prettierrc:5]: Yes, it gives quite less changes. I will update it. 
image

**Comment by l5io** [comment]
🚀 Preview for commit e97440cd85879363c4ee03bd9ae244b4de21809c at: https://669c6befec49536b9cb3313e--layer5.netlify.app

File: .prettierrc (hunk 1 above)
Comment (dottharun) [reply .prettierrc]: Request change is done.

**Comment by leecalcote** [comment]
@dottharun is this needed any longer?

**Comment by dottharun** [comment]
> @dottharun is this needed any longer?

This PR is for the prettier change (formatting) - the other PR #5680 #5719 was for 
eslint config (logic) only.

**Comment by dottharun** [comment]
> @dottharun merge conflicts

Fixed it.

**Comment by l5io** [comment]
🚀 Preview for commit f41208214b231472609b62080eb0a8252cfad845 at: https://66a503f6b95c2e2793c98724--layer5.netlify.app

**Comment by sudhanshutech** [comment]
@dottharun ?

**Pull request closed without merging**